wordpress-mobile/WordPress-Android
# Jetpack speed up settings

Fixes #7196 
Add "Speed up" settings to Site settings. There are two options with descriptions. One is to load images lazily and the second to server images from our servers. Both have been added to the "General" tab. Settings are only visible on Jetpack version >= 5.8

To test:

* Log in with site connected to Jetpack with version >= 5.8
* Go to site settings
* Click on "Speed up your site" 
* See 2 new settings with description
  * Both are clickable and are saved when clicked

![screenshot_1519051194](https://user-images.githubusercontent.com/1079756/36385908-53f1b210-1594-11e8-8256-848d5afccdff.png)
